pytential/linalg/utils.py

```diff
@@ -337,11 +340,21 @@ def interp_decomp(
     if rank is not None and eps is not None:
         raise ValueError("providing both 'rank' and 'eps' is not supported")
 
+    from scipy import __version__
+
+    kwargs = {}
+    if __version__ >= "1.15.0":
+        if rng is None:
+            rng = np.random.default_rng()
+
+        kwargs["rng"] = rng
```

Comment:
Had to start using `rng` in here because `scipy.linalg.interpolative.seed` is a noop on the latest version, so the tests kept failing due to randomness.